package-lock.json

```diff
@@ -4795,12 +4803,13 @@
       }
     },
     "node_modules/hypercore-crypto": {
-      "version": "3.4.0",
-      "license": "MIT",
+      "version": "3.4.2",
+      "resolved": "https://registry.npmjs.org/hypercore-crypto/-/hypercore-crypto-3.4.2.tgz",
+      "integrity": "sha512-16ii4M6T1dFfRa41Szv3IR0wXfImJMYJ8ysZEGwHEDH7sMeWVEBck6tg1GCNutYl39E+H7wMY2p3ndCRfj+XdQ==",
       "dependencies": {
-        "b4a": "^1.1.0",
-        "compact-encoding": "^2.5.1",
-        "sodium-universal": "^4.0.0"
+        "b4a": "^1.6.6",
+        "compact-encoding": "^2.15.0",
+        "sodium-universal": "^4.0.1"
```

Comment:
wondering if them changing `sodium-javascript` to a peer dep affects us? maybe not at this level but at the app level?

https://github.com/sodium-friends/sodium-universal/commit/44c1d6d5db3373e95abb707ed06db08160c4f99f

Reply:
Hmm, not sure. I'll try to test this out when I get a chance.

Reply:
yeah we are, so probably not an issue for us specifically. still a subtle breaking change otherwise 😄